JSDemos/Demos/DataGrid/StatePersistence/description.md

```diff
@@ -1 +1,5 @@
-Sort and filter data, group, reorder and resize columns, change page numbers and page size. Once you are done, refresh the web page to&nbsp;see that the grid&rsquo;s state is&nbsp;automatically persisted to&nbsp;continue working from where you stopped.
\ No newline at end of file
+**DataGrid** state can be persistent - if a user sorts and filters data, groups, reorders and resizes columns, or makes other changes, the widget saves these modifications and restores them when a user reloads the page. Refer to the [stateStoring](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/) article for a full list of modifications that can be saved.
```

Comment:
I'd change "state can be persistent" to "can persist its state". Better use a strong verb instead of any version of "to be".

 ```suggestion
**DataGrid** can persist its state. If a user sorts and filters data, groups, reorders and resizes columns, or makes other changes, the widget saves these modifications and restores them when a user reloads the page. Refer to the [stateStoring](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/) article for a full list of modifications that can be saved.
```